introduction/introduction.md

```diff
@@ -0,0 +1,13 @@
+This file holds the text to accompany the [introduction](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321606) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# Introduction
+
+This Learning Path gives an introduction to inner source.
+Inner source is the strategy of applying open source practices and principles to internal software development.
```

Comment:
> Inner source is the strategy of ...

We have always coined InnerSource as being _the application of_ open source practices and principles. I propose we stick with that.

Reply:
Done.